src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name,
+    age,
+    sex,
+    isMarried,
+    partnerName } = person;
+  const partnerDesignation = sex === 'm'
```

Comment:
```suggestion
  const partner = sex === 'm'
```
You can simplify the variable's name